mac/test/EnglishSpanish/README.md

```diff
@@ -0,0 +1,12 @@
+# EnglishSpanish test keyboard
+
+This keyboard is used only to test the crash reporting.
+
+1. In Keyman, turn on verbose logging.
+2. Optionally, in Console filter all messages from the **Keyman** process having the text **Sentry**
+3. Select a keyboard called **EnglishSpanish** (see /mac/test/EnglishSpanish folder in the source repo)
+4. Open an editable file in Xcode.
+5. Type **Sentry force now**
+   - With each keystroke, Console messages should indicate that the character is being added to the Easter egg string.
+   - When the final **w** is typed, a Console message should indicate: **Forcing crash now!**
```

Comment:
What PR is the code responsible for this in?

Reply:
This was done a long time ago -- I made a tweak to change the text to `Sentry force now` last week in [#3886](https://github.com/keymanapp/keyman/pull/3886/files#diff-865fd596e161ab2b5ddde99908a861bc6f46cce9ce1c2b8800175a0b0803774bR24)